consensus/src/pipeline/commit_reliable_broadcast.rs

```diff
@@ -36,8 +36,22 @@ impl CommitMessage {
     /// Verify the signatures on the message
     pub fn verify(&self, verifier: &ValidatorVerifier) -> anyhow::Result<()> {
         match self {
-            CommitMessage::Vote(vote) => vote.verify(verifier),
-            CommitMessage::Decision(decision) => decision.verify(verifier),
+            CommitMessage::Vote(vote) => {
+                let start_time = Instant::now();
```

Comment:
doesn't look like you need to `observe()` a manually measured duration in this case. just do 
```
{
    let _timer = counters::VERIFY_MSG.timer_with(&[commit_vote]);
    vote.verify(verifier)
}
```

Reply:
Done.

Reply:
saves two lines if you use `.timer_with()`, but your choice 😁